src/App.js

```diff
@@ -1,16 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import './App.css';
 // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- Kindly remove this commented piece of code, it makes your code not maintainable and clean.